src/components/navbar.component.tsx

```diff
@@ -78,6 +79,19 @@ const Navbar: React.FC<NavbarProps> = ({ user, onLogout, allowedLocales }) => {
               <Logo />
             </HeaderLink>
             <HeaderGlobalBar>
+              <HeaderGlobalAction
```

Comment:
Should be an extension slot (e.g., `top-nav-actions`). One extension to display here comes from `esm-patient-registration`. Also note that your link is not correct. It is hardcoded to the ref app and has a fully qualified URL, which will lead to a full page reload. You *must* use SPA navigation.

Reply:
Hello @FlorianRappl,
I am new to MF and this is my first PR. If possible can you explain the requirement in detail and share some resources which could help me understand what is the actual requirement to fix this.
Thank you.

Reply:
Hi @ujjwaltyagi355 - sure. Have you read the comment in the JIRA story at all?

> Sure Ujjwal Tyagi : Instead of having a hard coded button in there (e.g., `<MyButtonComponent />`) you should actually (and for all nav / menu buttons there) have an extension slot (e.g., `<ExtensionSlot extensionSlotName="nav-actions" />`). Now registered extension components (such as the add registration button) would show up automatically. The order of these extension components would be done via configuration, where we set the + button by default to the desired location as shown in the wireframes.

> The actual extension component of this story (add registration button) would be part of `@openmrs/esm-patient-registration`.

What is unclear? Have you watched the videos yet?

https://github.com/openmrs/openmrs-esm-core/blob/master/docs/VIDEOS.md

Specifically, the video introducing the extension system might be interesting for you. https://www.youtube.com/watch?v=crdEL91oBGs&feature=youtu.be

Reply:
@FlorianRappl, I am really sorry for that. 
I just implemented this before the update in the requirements. So, eventually, when I looked at your comment I was not able to understand much, I tried to find out about "Extension" in React framework as well as Carbon. So, I thought it was a term to describe the way other buttons were laid out in the navbar like "Location" and "user".  So I did the same with the "Add user" button.
So, now when you commented the same here, I just thought maybe I just need to ask. 
I sincerely apologize for this, and I have started to work upon the videos you just shared.
Thank you for the help.